specs/protocol/abi.md

```diff
@@ -687,3 +687,31 @@ Calling `bar` with `MySumType::Z` yields:
 0x
 0000000000000002 // The discriminant of the chosen enum, in this case `2`.
 ```
+
+## JSON Format for Storage Initializers
+
+Contracts can request that certain storage slots are initialized to specific values. These initialized slots are represented in JSON format as an array where each element represents a storage slot and has the following properties:
+
+- `"key"`: String, a 32-byte key for a given storage slot;
+- `"initial_value"`: String, a 32-byte value that initializes the slot;
```

Comment:
Since this is initializers, the fact that it's initial is implicit. Could shorter to just `value`.